pkg/consensus/prepare_request.go

```diff
@@ -12,7 +12,7 @@ type prepareRequest struct {
 	timestamp         uint64
 	nonce             uint64
 	transactionHashes []util.Uint256
-	minerTx           transaction.Transaction
+	tx                transaction.Transaction
```

Comment:
Take a look at C# code, it doesn't have any `tx` in its `PrepareRequest` and there is no need for it.